data/default_release_images.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "openshift_version": "4.6",
```

Comment:
I believe this new `releases` json is to support different cluster versions than OCP like OKD.
So maybe this field should be named `version` and have an additional `type` field for `openshift` value?

Reply:
Actually the main incentive is for an easier multiple architecture support, i.e. having a simple list as os images json instead of a complex structure in openshift_versions. So since we already have a 'version' property for the release and to keep the object simple, we could just add okd_version property later on if required.